src/lib/coda_intf/network_intf.ml

```diff
@@ -43,10 +43,10 @@ module type Network_intf = sig
        Proof_carrying_data.t
        Deferred.Or_error.t
 
-  val get_bootstrappable_best_tip :
+  val get_best_tip :
        t
     -> Network_peer.Peer.t
-    -> Consensus.Data.Consensus_state.Value.t
+    -> unit
```

Comment:
please don't make this a unit

Reply:
What do you mean by not make this a unit?